lib/travis/api/v3/router.rb

```diff
@@ -12,6 +12,8 @@ def initialize(routes = Routes)
     end
 
     def call(env)
+      Metriks.meter("api.v3.total_requests").mark
```

Comment:
I looked for another way to get this metric from librato already, but can't find one, so all LGTM